src/Extensions/DependencyInjection/FunctionHostBuilderContext.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.Azure.Functions.Extensions.DependencyInjection
+{
+    public abstract class FunctionHostBuilderContext
+    {
+        private string _rootPath;
+
+        public string RootPath
```

Comment:
nit: I was calling this `ApplicationRootPath` but I also know that it's called `ContentRootPath` in [IHostingEnvironment](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.hosting.ihostingenvironment.contentrootpath?view=aspnetcore-3.1#Microsoft_AspNetCore_Hosting_IHostingEnvironment_ContentRootPath). Should we pick one of those to be a little more specific?

Reply:
I was going with the `FunctionsContext.RootPath`approach, but `ApplicationRootPath` makes sense.